build.gradle

```diff
@@ -21,18 +21,17 @@
 buildscript {
     repositories {
         jcenter()
+        google()
     }
     dependencies {
-        classpath 'com.android.tools.build:gradle:2.3.3'
+        classpath 'com.android.tools.build:gradle:3.0.1'
```

Comment:
If the gradle version is upgraded, the output of the debug *.apk changes. There will be an additional subdirectory under `build/outputs/apk/` for the build flavor (eg debug). Please respect the new Location in the `Jenkinsfile`. It is defined inside the `APK_LOCATION_DEBUG` variable. It should change from `.../apk/Paintroid-debug.apk` to `.../apk/debug/Paintroid-debug.apk`

Reply:
Should be fixed now